src/tilesource.js

```diff
@@ -191,7 +196,11 @@ $.TileSource = function( width, height, tileSize, tileOverlap, minLevel, maxLeve
         this.aspectRatio = ( options.width && options.height ) ?
             (  options.width / options.height ) : 1;
         this.dimensions  = new $.Point( options.width, options.height );
-        this.tileSize    = options.tileSize ? options.tileSize : 0;
+        
+        this.tileSize = options.tileSize ? options.tileSize : 0;
```

Comment:
Since TileSource can support an array of arguments as well as an object, should we worry about supporting tileWidth and tileHeight there as well? 

https://github.com/openseadragon/openseadragon/blob/master/src/tilesource.js#L93-100